low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view

```diff
@@ -71,7 +71,7 @@ class SSPLHealthView(object):
     encl_controllers_filename = storage_encl_dir+'controllers/controllerdata.json'
     encl_fans_filename = storage_encl_dir+'fanmodules/fanmodule_data.json'
     encl_sideplane_exp_filename = storage_encl_dir+'sideplane_expanders/sideplane_expanders_data.json'
-    encl_logical_volumes_filename = storage_encl_dir+'logical_volumes/logicalvolumedata.json'
+    encl_logical_volumes_filename = storage_encl_dir+'logical_volumes/logical_volume_alert_data.json'
```

Comment:
I changed the key name because if we use the same key, it can lead to issues on setups where old data is present (SW upgrade). We can rename the new key to "logical_volume_data" if required.